Roles/Crewmate/Retributionist.cs

```diff
@@ -120,6 +120,9 @@ public static bool RetributionistMsgCheck(PlayerControl pc, string msg, bool isU
         try
         {
             targetId = int.Parse(msg.Replace("/ret", string.Empty));
+            targetId = int.Parse(msg.Replace("/惩罚", string.Empty));
+            targetId = int.Parse(msg.Replace("/惩", string.Empty));
+            targetId = int.Parse(msg.Replace("/罚", string.Empty));
```

Comment:
Huh, this could probably cause problems.
But I'll write code later on how best to do this